vdom/src/com/vdom/core/Player.java

```diff
@@ -24,6 +24,7 @@
     public static final String THREE_PLUS_COPY_ACTION_CARDS = "Three Plus Copy Action Cards";
     public static final String NON_VICTORY_EMPTY_SUPPLY_PILE_CARDS = "Non Victory Empty Supply Pile Cards";
     public static final String SECOND_MOST_COMMON_ACTION_CARDS = "Second Most Common Action Cards";
+    public static final String CASTLE_CARDS = "Castle Cards";
```

Comment:
I've now made it so that it Displays a total and below a list of the castles the player has: